t/test.py

```diff
@@ -373,6 +394,14 @@ def test_strptime(self):
 
         self.assertEqual(True, dt1 == dt2)
 
+    def test_strptime_persian(self):
+        date_string = u"۱۳۶۳-۶-۶ 1۲:۱3:۱۴"
```

Comment:
Not sure if intentional, but the characters are mixed in the sample time:
```
۱۳۶۳-۶-۶ 1۲:۱3:۱۴
```
(`1` and `3` are latin numerals).
May I suggest to have different test cases, to help document the feature? one test case with all characters in Persian numerals, and another for mixed numerals.

Reply:
done